lib/scraped_page_archive.rb

```diff
@@ -14,41 +14,100 @@
 class ScrapedPageArchive
   class Error < StandardError; end
 
-  attr_writer :github_repo_url
+  class GitStorage
+    attr_writer :github_repo_url
```

Comment:
Ah, no it's not going to work now. I've updated the README in https://github.com/everypolitician/scraped_page_archive/commit/b6b8f8664d2031777bb870d145093a35d5cef408, the interface I've documented is quite long winded, but I think based on past experience that's probably best to start with, then we can add a shortcut in a future PR.